src/components/pages/donation_form/SubmitValues.vue

```diff
@@ -49,16 +49,16 @@ export default Vue.extend( {
 	},
 	computed: {
 		...mapState( NS_PAYMENT, {
-			payment: state => ( state as Payment ).values,
+			payment: ( state: any ) => ( state as Payment ).values,
```

Comment:
`any` should be used sparingly, if there is no other option. in case of `SubmitValues`, we can avoid this as follows: 
```typescript
...mapState<Payment>( NS_PAYMENT, {
    payment: (state: Payment) => payment.value,
// etc
```

Could you please change the code?